tslint.json

```diff
@@ -1,9 +1,52 @@
 {
     "defaultSeverity": "error",
     "extends": [
-        "tslint:recommended"
+      "tslint:recommended",
+      "tslint-react"
     ],
+    "linterOptions": {
+      "exclude": [
+        "node_modules/**",
+        "typings/**",
+        "App/Sagas/**/*",
```

Comment:
Not sure what order you created the two branches for this lint work, but this looks weird to me. My impression is that this branch should be linting all files. Not sure why these exclude items are here.

Reply:
FYI... I did already merge the other lint branch into this one.

Reply:
I put those in there on purpose. It was so I didn't have to clean up all the tslint issues in those folders that you are changing so much in your branch. It was a best attempt to avoid merge hell. Then we can go back and selectively add them.

Reply:
Ah ok cool. I thought that was the case in the other branch and this branch was to go all in. This works, tho

Reply:
yeah... would be a nightmare